integrationtest/backend-lucene/src/test/java/org/hibernate/search/v6poc/integrationtest/backend/lucene/ExtensionIT.java

```diff
@@ -226,30 +236,115 @@ public void sort_fromJsonString_separateSort() {
 				.hasReferencesHitsExactOrder( indexName, THIRD_ID, SECOND_ID, FIRST_ID, FOURTH_ID, FIFTH_ID );
 	}
 
+	@Test
+	public void predicate_customField_throwsException() {
```

Comment:
Maybe use the term "native" rather than "custom"? All fields are "custom" in some way, after all, even non-native ones.